src/applications/vaos/appointment-list/redux/selectors.js

```diff
@@ -115,6 +118,23 @@ export const selectFutureAppointments = createSelector(
   },
 );
 
+export const selectUpcomingAppointments = createSelector(
+  state => state.appointments.pending,
```

Comment:
May want to add a comment here making note that we are grabbing pending only for Express Care appts.